src/registrar/tests/test_admin_request.py

```diff
@@ -160,7 +160,6 @@ def test_helper_text(self):
             ("no_other_contacts_rationale", "Required if creator does not list other employees"),
             ("alternative_domains", "Other domain names the creator provided for consideration"),
             ("no_other_contacts_rationale", "Required if creator does not list other employees"),
-            ("Urbanization", "Required for Puerto Rico only"),
         ]
```

Comment:
Removed this since we are getting urbanization from portfolio, and this text is readonly

Reply:
@asaki222 you mean read-only if the request is in a portfolio right? not read-only all the time

Reply:
@abroddrick I missed this!  It makes sense for it to be editable for non portfolio domain requests. I will update.